tests/test_end_to_end_0.py

```diff
@@ -1,24 +1,21 @@
-# from biasdetector import Evaluator
 from ai_biases_analyzer.evaluator import Evaluator
 from ai_biases_analyzer.model import Model_SD_0
 from ai_biases_analyzer.plot import plot_chart_genderrace
 
 
 def test_instantiate_evaluator():
     """Instantiating evaluator shouldn't crash"""
-
     Evaluator()
 
 
 def test_instantiate_model():
     """Instantiating model shouldn't crash"""
-
     Model_SD_0()
 
 
 def test_e2e_0():
     """Test meant to ensure that the end-to-end pipeline doesn't crash for this plot"""
-
+    # REFACTOR: This test is not e2e, generator is not tested.
```

Comment:
Correct!! Although the generator might prove a bit challenging to test since it requires a back-end to run the AI model, but maybe we could setup a mock server to ensure the rest of the functionality runs!